Automattic/jetpack
# Add: Search

Fixes #4638 

**Currently depends on linking with https://github.com/Automattic/dops-components/pull/41**

![search-demo](https://cloud.githubusercontent.com/assets/7129409/17517442/75366e48-5e11-11e6-94c6-b5a1c7106e5d.gif)

**To Test:** 
- Clicking the search icon should take you to `/search` route.  You should see a list of all modules. 
- Make sure you can activate/deactivate the modules
- Make sure you can expand/configure the settings of some modules 
- Search for any term.  The search terms are pulled in from the module headers. 
- Clicking the `X` to close search should bring you `back` to the route you were on previously. 
- Make sure that the PRO cards display correct information 
- Make sure the PRO DashItems still display correctly 
- Search terms should be translatable
- Dev Mode should be represented properly 
- Non-admins should not be able to see the search icon at all
- The sub-nav wp-admin menu should be styled properly (Settings item highlighted)
